packages/serverless/src/apollo/app/proof.ts

```diff
@@ -95,11 +102,39 @@ const getProof = publicWrapper(
   }
 );
 
+const getDatabaseProofStatus = publicWrapper(
+  Joi.object({
+    databaseName,
+  }),
+  async (_root: unknown, args: TProofRequest) => {
+    const modelTask = ModelQueueTask.getInstance();
+
+    const task = await modelTask.findOne({
+      database: args.databaseName,
+      status: { $in: ['proving', 'queued'] },
+    });
+
+    if (task) {
+      return 'EMPTY';
+    } else {
+      const modelProof = ModelProof.getInstance();
+      const proof = await modelProof.getProof(args.databaseName);
+
+      if (proof) {
+        return 'PROVED';
+      } else {
+        return 'PENDING';
+      }
```

Comment:
using ternary operator more cleaner. Also I think you should add it to typescript Enum
```ts
return proof ? 'PROVED' : 'PENDING'
```

Reply:
done